features/manage_meeting_rooms.feature

```diff
@@ -4,11 +4,23 @@ Feature: Manage Meeting Rooms
   I want to create and delete meeting rooms
 
   Scenario: Create Meeting Room
-    Given A POST request is made
-    When admin create a meeting room
-    And I fill in 'name' with 'Gyan'
-    And I fill in 'capacity' with '50'
-    Then I should see status '201'
-    And I should see 'Gyan'
-    And I should have 1 meeting room
+    When I send a POST request to "/v1/meeting_rooms" with the following:
+      | name | gyan |
+      | capacity   | '100' |
+    Then the response status should be "201"
```

Comment:
Good you are using `201` response code here. Is there a response body as well that you can test?

Reply:
yes, we are returning JSON as well, I'm adding that scenario as well in which we can test both status and JSON

Reply:
Done

Reply:
I still feel like the steps sounds too technical and Business owners won't be able to understand things like 201, end point path '/v1/meeting_rooms'. May be we can still simplify the steps description as below:

Given As a Manager
When I create a meeting room with name 'Test'
Then I should see a meeting room created successfully 

@glen please correct me if I am wrong here. @kirti09